src/Core_Command.php

```diff
@@ -564,28 +570,37 @@ function wp_new_blog_notification() {
 				}
 			}
 			// WP 4.9.0 - skip "Notice of Admin Email Change" email as well (https://core.trac.wordpress.org/ticket/39117).
-			add_filter( 'send_site_admin_email_change_email', function() {
-				return false;
-			} );
+			add_filter(
+				'send_site_admin_email_change_email',
+				function() {
+					return false;
+				}
+			);
 		}
 
-		require_once( ABSPATH . 'wp-admin/includes/upgrade.php' );
+		require_once ABSPATH . 'wp-admin/includes/upgrade.php';
 
-		extract( wp_parse_args( $assoc_args, array(
-			'title' => '',
-			'admin_user' => '',
-			'admin_email' => '',
-			'admin_password' => ''
-		) ), EXTR_SKIP );
+		extract(
```

Comment:
Suggestion for readability - move the defaults array out of the function call.

```php
		$defaults = array(
			'title'          => '',
			'admin_user'     => '',
			'admin_email'    => '',
			'admin_password' => '',
		);
		extract( wp_parse_args( $assoc_args, $defaults ), EXTR_SKIP );
```

Also: using `extract()` is strongly discouraged, so if you can avoid it/refactor, that would be recommended.